src/pages/SettingsPage.tsx

```diff
@@ -1,7 +1,189 @@
-import React from "react";
+import React, { useState } from "react";
+import {
+  Radio,
+  RadioGroup,
+  FormControlLabel,
+  FormControl,
+  FormLabel,
+  Box,
+  Grid,
+  InputAdornment,
+  IconButton,
+  Input,
+  Button,
+} from "@material-ui/core";
+import { Backspace, LocalPhone } from "@material-ui/icons";
 import { Header } from "components/Header";
 import { labels } from "labels";
 
+interface Options {
+  value: string;
+  label: string;
+}
+interface SettingsRadioGroupProps {
+  formLabel: string;
+  name: string;
+  options: Options[];
+  value: string;
+  setValue: (to: string) => void;
+}
+interface SettingKeyPadProps {
+  buttonsValues: string[];
+  number: string;
+  setNumber: (newNum: any) => void;
+}
+const SettingsRadioGroup: React.FC<SettingsRadioGroupProps> = ({
```

Comment:
This file is getting too big. Please move these two subcomponents into their own separate files